integration-tests/tx-query2-enclave-app.sgxs

```diff
@@ -0,0 +1 @@
+../target/x86_64-fortanix-unknown-sgx/debug/tx-query2-enclave-app.sgxs
```

Comment:
what are these paths in files for?

Reply:
oh, the file created after I run the integration test, I'll remove it.